tests/foreman/maintain/test_health.py

```diff
@@ -0,0 +1,882 @@
+
+    :expectedresults: Health check should perform.
+    """
+    result = sat_maintain.cli.Health.check(options={'label': 'available-space'})
+    assert 'FAIL' not in result.stdout
+    assert result.status == 0
+
+
+def test_positive_available_space_candlepin(sat_maintain):
+    """Verify available-space-cp check
+
+    :id: 382a2bf3-a3da-4e46-b370-a443450f93b7
+
+    :steps:
+        1. Run satellite-maintain health check --label available-space-cp
+
+    :expectedresults: Health check should perform.
+
+    :CaseImportance: Medium
+    """
+    result = sat_maintain.cli.Health.check(options={'label': 'available-space-cp'})
+    assert 'FAIL' not in result.stdout
+    assert result.status == 0
+
+
+def test_positive_hammer_defaults_set(sat_maintain, request):
+    """Verify that health check is performed when
+     hammer on system have defaults set.
+
+    :id: 27a8b49b-8cb8-4004-ba41-36ed084c4740
+
+    :steps:
+        1. Setup hammer on system with defaults set
+        2. Run satellite-maintain health check
+
+    :expectedresults: Health check should perform.
+
+    :BZ: 1632768
+    """
+    sat_maintain.cli.Defaults.add({'param-name': 'organization_id', 'param-value': 1})
+    result = sat_maintain.cli.Health.check(options={'assumeyes': True})
+    assert result.status == 0
+    assert 'FAIL' not in result.stdout
+
+    @request.addfinalizer
+    def _finalize():
+        sat_maintain.cli.Defaults.delete({'param-name': 'organization_id'})
+
+
+@pytest.mark.include_capsule
+def test_positive_check_hotfix_installed(sat_maintain, request):
+    """Verify check-hotfix-installed check.
+
+    :id: d9023293-4173-4223-bbf5-328b41cf87cd
+
+    :parametrized: yes
+
+    :steps:
+        1. Run satellite-maintain health check --label check-hotfix-installed
+
+    :expectedresults: check-hotfix-installed check should detect modified file
+        and installed hotfix.
+    """
+    gems_path = '/usr/share/gems/gems/'
+    fpath = sat_maintain.execute(f"find {gems_path} -type f -name 'fog-vsphere*'")
+    sat_maintain.execute(f'sed -i "$ a #modifying_file" {fpath.stdout}')
+    hotfix_url = f'{settings.robottelo.repos_hosting_url}/hotfix_package/'
+    sat_maintain.create_custom_repos(custom_repo=hotfix_url)
+    result = sat_maintain.cli.Packages.unlock()
+    assert 'FAIL' not in result.stdout
+    assert result.status == 0
+    result = sat_maintain.execute('dnf install hotfix-package -y')
+    assert 'FAIL' not in result.stdout
+    assert result.status == 0
+    result = sat_maintain.cli.Health.check(
+        options={'label': 'check-hotfix-installed', 'whitelist': 'check-non-redhat-repository'}
+    )
+    assert 'WARNING' in result.stdout
+    assert 'hotfix-package' in result.stdout
+    # assert setup_hotfix_check in result.stdout
+    assert result.status == 78
+
+    @request.addfinalizer
+    def _finalize():
+        assert sat_maintain.execute(f'sed -i "/#modifying_file/d" {fpath.stdout}').status == 0
```

Comment:
```
[D 221115 13:08:59 hosts:92] ip-10-0-218-255.rhos-01.prod.psi.rdu2.redhat.com executing command: sed -i "/#modifying_file/d" 
[D 221115 13:08:59 hosts:94] ip-10-0-218-255.rhos-01.prod.psi.rdu2.redhat.com command result:
    stdout:
    
    stderr:
    (20, b'sed: no input files\n')
    status: 4
```
sed is not recognizing `{fpath.stdout}` as a valid path, so this assertion fails and the following commands are not run. This is causing a failure on `test_positive_check_hotfix_installed_without_hotfix` because `hotfix-package` is not being removed.

Reply:
There was a path change which was not updated
Updated now.